grapple/actions.py

```diff
@@ -452,6 +456,8 @@ class Meta:
             ]
         else:
             interfaces = (interface,) if interface is not None else ()
+        # Add description to type if streamfield has a docstring
+        description = cls.__doc__
```

Comment:
It should be possible to also depreciate streamfield blocks, I looked into expanding the `register_streamfield_block` decorator to support passing in an additonal argument however, I couldn't seem to find a way to make this work in `build_streamfield_type`. 
As a compromise I thought about using the blocks docstring instead, but happy to remove it if not ideal?

Reply:
@Morsey187 if you can update the comment with the extra context (and nitpick "streamfield" -> "the block")

Reply:
What if, instead of implicitly including a block's docstring as the description, we check the block's metaclass for the presence of a `graphql_description` field? Developers may not want their internal documentation exposed.

Reply:
Implemented in https://github.com/torchbox/wagtail-grapple/pull/333/commits/4f006511bc70395df850093226563b6a9302221f

Reply:
I added docs under `register_streamfield_block`. Not sure what to add to the comment in code from Ben's comment above, or if it's still necessary?

Reply:
I think things are good as they are now. Thanks both